playbooks/Splunk_Attack_Analyzer_Dynamic_Analysis.json

```diff
@@ -263,17 +263,17 @@
                             "join": [],
                             "note": "Queries SAA for information about the provided vault_id(s)"
                         },
-                        "connector": "Splunk Attack Analyzer",
+                        "connector": "Splunk Attack Analyzer Connector for Splunk SOAR",
                         "connectorConfigs": [
-                            "splunk_attack_analyzer"
+                            "splunk attack analyzer app"
```

Comment:
@kelby-shelton : shouldn't this have underscores?

Reply:
It doesn't actually matter because the user will be prompted to map it to their own asset name before they save it locally.

Reply:
sweeet! thanks for the clarification. We will release this in 4.12.0